.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.*
+  workflow_dispatch:
```

Comment:
Use `workflow_call` only here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call

We don't want to run on tags because then we'll get duplicate runs.

Reply:
Workflows don't trigger off events happening in github actions, so there wouldn't be any duplicates

Reply:
Neat, TIL! Even so, if we use `workflow_call` we still don't need this to run on tags.

Reply:
> Workflows don't trigger off events happening in github actions, so there wouldn't be any duplicates

Just FYI: They can if you use a Personal Access Token.